cmd/checkmarxExecuteScan.go

```diff
@@ -522,7 +547,7 @@ func getDetailedResults(sys checkmarx.System, reportFileName string, scanID int)
 		return resultMap, errors.Wrap(err, "failed to download xml report")
 	}
 	if len(data) > 0 {
-		ioutil.WriteFile(reportFileName, data, 0700)
+		_ = ioutil.WriteFile(reportFileName, data, 0700)
```

Comment:
Basically the same like already mentioned above for `zipFolder`

Reply:
done